vickev/howdypix
# Add graphql endpoint

## Description

Add GraphQL endpoints!





## How to test?

- Check-out this branch.
- `yarn dev`
- Go to `http://localhost:4005/` and type the following query:

```
# Write your query or mutation here
{
  getAlbum(parent: 1) {
    photos {
      thumbnails
    }
    albums {
      id
      name
    }
  }
}
```

## Checklist
